.circleci/config.yml

```diff
@@ -141,6 +141,26 @@ jobs:
           name: Upload Coverage
           command: bash <(curl -s https://codecov.io/bash) -cF python
 
+  build_image:
+    docker: 
+      - image: docker
+    steps:
+      - setup_remote_docker
+      - checkout
+      - run:
+          name: Docker Build
+          command: docker build -t prefecthq/prefect .
+      - run:
+          name: Test image
+          command: |
+            docker run -dit prefecthq/prefect /bin/bash -c 'curl -fL0 https://raw.githubusercontent.com/PrefectHQ/prefect/master/examples/retries_with_mapping.py | python'
+      - run:
+          name: Authenticate with Docker Hub and push
+          command: |
+            docker login --username  prefectci --password $DOCKER_HUB_PW
```

Comment:
I would make the username an environment variable as well

Reply:
we need to come up with a solution to the issue with the secret env vars before I figure out what this will look like

Reply:
what issue are you referring to @TylerWanner ?

Reply:
fixed